### PR TITLE
SITES-523 Incorrect dates in TAS and QLD dates and time content

### PR DESCRIPTION
**Pull request by maxious**
SITES-523 Incorrect dates in TAS and QLD dates and time content

### CODE REVIEW

File: app/views/templates/custom/school_holidays_tas.html.erb [hunk 1]
```diff
@@ -107,7 +107,7 @@
       <h3>
         Term 1 <span></span>
       </h3>
-      <p><time datetime="2017-02-087">Wednesday 8 February</time> to <time datetime="2017-04-137">Thursday 13 April</time> <span>Term 1 includes the Easter break, Friday 14 April to Tuesday 18 April.</span></p>
+      <p><time datetime="2017-02-087">Wednesday 8 February</time> to <time datetime="2017-04-137">Thursday 13 April</time></p>
```
Comment (tessereth) [inline app/views/templates/custom/school_holidays_tas.html.erb]: The 87th February?

File: app/views/templates/custom/school_holidays_tas.html.erb [hunk 2]
```diff
@@ -116,7 +116,7 @@
       <h3>
         Autumn holidays
       </h3>
-      <p><time datetime="2017-04-117">Monday 11 April</time> to <time datetime="2017-04-257">Monday 25 April</time></p>
+      <p><time datetime="2017-04-14">Friday 14 April</time> to <time datetime="2017-04-257">Monday 25 April</time></p>
```
Comment (tessereth) [inline app/views/templates/custom/school_holidays_tas.html.erb]: 257th April?

**Comment by maxious** [comment]
Fixed invalid date attributes

**Comment by tessereth** [comment]
LGTM

**Comment by bjgordon** [comment]
LGTM

**Pull request merged**